src/main/scala/nl/tradecloud/kafka/KafkaSubscriber.scala

```diff
@@ -25,7 +25,7 @@ class KafkaSubscriber(
     batchingSize: Int = 1,
     batchingInterval: FiniteDuration = 3.seconds,
     system: ActorSystem,
-    offset: ConsumerOffset.Value
+    initialOffset: Option[ConsumerOffset.Value] = None
```

Comment:
please rename initialOffset to autoOffsetReset (equivalent as the Kafka consumer setting) as initial is vague, eg. its only when there is no offset available

Reply:
and rename ConsumerOffset to AutoOffsetReset